rel-eng/openshift-master.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=OpenShift Master
+Documentation=https://github.com/openshift/origin
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/sysconfig/openshift-master
+ExecStart=/usr/bin/openshift start $ROLE $OPTIONS
+WorkingDirectory=/var/lib/openshift/
```

Comment:
Yeah, /var/lib/openshift is created by the base 'openshift' package. Are you suggesting that 'openshift-master' package creates and uses its own working directory?